lib/common/Makefile.am

```diff
@@ -35,7 +35,7 @@ libcrmcommon_la_LDFLAGS	= -version-info 47:1:13
 libcrmcommon_la_CFLAGS	= $(CFLAGS_HARDENED_LIB)
 libcrmcommon_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
 
-libcrmcommon_la_LIBADD = @LIBADD_DL@
+libcrmcommon_la_LIBADD = @DL_LIBS@
```

Comment:
This was wrong to start with ... libstonithd uses dlopen(), not libcrmcommon

Reply:
I haven't dug into it too far because I don't want to become the autoconf expert, but I'm not sure that all this checking for libdl even matters given that we use `LT_INIT([dlopen])`.

Reply:
It looks like `LT_INIT([dlopen])` is important for letting libtool know we might use dlopen() (though I'm not positive; it might only be needed for building libraries that might get dlopen()'ed themselves). I don't think it *requires* dlopen, and I'm not sure whether it sets up LIBS or not.

But since we had LIBADD_DL in the wrong library anyway, we're clearly not using it. Feel free to either keep the search and use it in the correct library, or just drop the search and use REQUIRE_FUNC.

Reply:
Updated to keep the search and do the substitution in the right file.